zqd/space/space.go

```diff
@@ -44,26 +40,25 @@ func newSpaceID() api.SpaceID {
 }
 
 type Space struct {
+	Storage *storage.ZngStorage
+
 	path string
 	conf config
-
-	index *zngio.TimeIndex
+	mu   sync.RWMutex
```

Comment:
Please remove the opMutex, deletePending, and other associated changes, if needed, those can happen in a separate PR.

Reply:
Sure.

Reply:
Not needed, but simpler with less moving parts.